command/delegate/delegate.go

```diff
@@ -318,6 +323,35 @@ func (c *delegateCommand) handleSetup(w http.ResponseWriter, r *http.Request) {
 		ctx = logger.WithContext(r.Context(), logger.Logrus(logr))
 	}
 
+	// load the configuration from the environment
+	env, err := fromEnviron()
+	if err != nil {
+		log.Errorf("could not load config from environment: %s", err)
+		return
+	}
+
+	// append global volumes to the setup request.
+	for _, pair := range env.Runner.Volumes {
+		plen := 2
```

Comment:
L335-L343 is used in 3 places. Let's create a method for it that probably returns src & ro & use it instead of duplicating the code.

Reply:
done